src/routes/news/constants.js

```diff
@@ -1,4 +1,6 @@
 // @flow
 
+export type NewselementType = 'LOCAL_NEWS' | 'TU_NEWS'
```

Comment:
```suggestion
export type NewsType = 'LOCAL_NEWS' | 'TU_NEWS'
```
should be enough. If you don't think so, change it to `NewsElementType` with a upper E.

Reply:
Fixed